app/uk/gov/hmrc/apidocumentation/config/ApplicationConfig.scala

```diff
@@ -120,6 +122,8 @@ class ApplicationConfigImpl @Inject()(config: Configuration, runMode: RunMode)
   val principalBaseUrl = getConfigDefaulted(s"$env.features.principalBaseUrl", "https://api.service.hmrc.gov.uk")
   val subordinateBaseUrl = getConfigDefaulted(s"$env.features.subordinateBaseUrl", "https://test-api.service.hmrc.gov.uk")
 
+  val platformFrontendHost = getString("platform.frontend.host")
```

Comment:
fyi: This is config from some global environment config which basically has this in it for each env:
https://www.qa.tax.service.gov.uk/